docs/radar.css

```diff
@@ -1,10 +1,11 @@
 body {
-  font-family: 'Source Sans Pro', arial, helvetica, sans-serif;
-  display: flex;
-  align-items: center;
-  flex-direction: column;
+  font-family: "Source Sans Pro", arial, helvetica, sans-serif;
   padding-bottom: 50px;
 }
+.container {
+  margin: auto;
+  width: 1450px;
```

Comment:
this is the actual change, most/all of the other things are related to prettier setup & reformatting code to conform with prettier defaults.